3rdparty/qxt/qxtglobalshortcut_x11.cpp

```diff
@@ -29,8 +29,11 @@
 ** <http://libqxt.org>  <foundation@libqxt.org>
 *****************************************************************************/
 
-#if QT_VERSION < QT_VERSION_CHECK(5,0,0)
+#if QT_VERSION < QT_VERSION_CHECK(5,0,0) || QT_VERSION > QT_VERSION_CHECK(5,6,0) 

```

Comment:
Need need to do this check at all, since Qt 5.6 is required. So just use QX11Info